### PR TITLE
Add initial support for extra OVN cluster per node for ovn-bgp-agent

### PR DESCRIPTION
**Pull request by luis5tb**

### CODE REVIEW

**Comment by luis5tb** [comment]
recheck seems to be an unrelated error

**Comment by luis5tb** [comment]
recheck nodefailure

**Comment by luis5tb** [comment]
recheck node failure

**Comment by luis5tb** [comment]
recheck unrelated as it failed before edpm-ansible

**Comment by luis5tb** [comment]
recheck node_failure

File: roles/edpm_ovn_bgp_agent/tasks/configure.yml
Comment (rebtoor) [inline roles/edpm_ovn_bgp_agent/tasks/configure.yml:26]: I've seen these tasks in, at least, 3 roles. I mean, they're correct, but maybe we could choose a role, move these tasks into a single tasks file and use `include_role/tasks_from` directives to avoid code duplication.

File: roles/edpm_ovn_bgp_agent/tasks/configure_ovn.yml
Comment (rebtoor) [inline roles/edpm_ovn_bgp_agent/tasks/configure_ovn.yml:25]: ditto :)

**Review by rebtoor — COMMENTED** [review]
i left a couple of comments to address, but i'm ok with this role. I'd wait for another couple of eyes before approving it.

**Review by fao89 — COMMENTED** [review]
+2

**Comment by luis5tb** [comment]
First we need https://github.com/openstack-k8s-operators/dataplane-operator/pull/539

**Comment by rebtoor** [comment]
@luis5tb can you squash your commits in a single commit so we can keep our history clean? Thanks!

**Comment by rebtoor** [comment]
> First we need https://github.com/openstack-k8s-operators/dataplane-operator/pull/539 

Please, add a `Depends-on` in the pr description and put this PR on hold (until the other pr will be merged) to avoid issues.

File: roles/edpm_ovn_bgp_agent/defaults/main.yml
Comment (karelyatin) [inline roles/edpm_ovn_bgp_agent/defaults/main.yml:34]: can use consistent name i.e edpm_ovn_bgp_agent_config_src

Comment (luis5tb) [reply roles/edpm_ovn_bgp_agent/defaults/main.yml:34]: ok!

**Comment by luis5tb** [comment]
> > First we need [openstack-k8s-operators/dataplane-operator#539](https://github.com/openstack-k8s-operators/dataplane-operator/pull/539)
> 
> Please, add a `Depends-on` in the pr description and put this PR on hold (until the other pr will be merged) to avoid issues.

done

**Comment by luis5tb** [comment]
retest dataplane-operator patch merged

**Comment by luis5tb** [comment]
recheck

**Review by fao89 — APPROVED** [review]

**Pull request merged**
